server/option.go

```diff
@@ -0,0 +1,14 @@
+package server
+
+import (
+	"github.com/openfresh/plasma/config"
+	"github.com/openfresh/plasma/pubsub"
+	"go.uber.org/zap"
+)
+
+type Option struct {
```

Comment:
__[golint]__
* [golint] exported type Option should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->